js/CRM_Contact_Page_View_UserDashBoard.js

```diff
@@ -17,5 +17,33 @@
     var html = $(el).html().replace(/[\[\]]/g , '');
     $(el).html(html);
   })
+
+  // Change renew button link to match the users current membership type
+  $(window).load(function(){
```

Comment:
Oops, just noticed this. Please use `$(document).ready` here. It runs  a little earlier than `load`, and `load` is being phased out, see https://stackoverflow.com/a/8396457

Reply:
Done fix